### DISCUSSION TITLE
Numpy not in the developer version

### DISCUSSION THREAD

**Discussion by wagdy88** [jonescompneurolab/hnn-core#595] [Installation]
Hi guys, 
I tried installing the developer version from the instructions in this link:
https://jonescompneurolab.github.io/hnn-core/dev/index.html

First, the installation stated the numpy was installed (or satisfied) - see below. 
pip install --upgrade https://api.github.com/repos/jonescompneurolab/hnn-core/zipball/master

`pip install --upgrade https://api.github.com/repos/jonescompneurolab/hnn-core/zipball/master

Defaulting to user installation because normal site-packages is not writeable
Collecting https://api.github.com/repos/jonescompneurolab/hnn-core/zipball/master
  Downloading https://api.github.com/repos/jonescompneurolab/hnn-core/zipball/master
     | 528.9 kB 3.2 MB/s 0:00:00
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Installing backend dependencies ... done
  Preparing metadata (pyproject.toml) ... done
Requirement already satisfied: numpy>=1.14 in /usr/lib/python3/dist-packages (from hnn-core==0.3.dev0) (1.21.5)
`

Then, when I tried to search for it conda list it didn't show up. 
`pip install --upgrade https://api.github.com/repos/jonescompneurolab/hnn-core/zipball/master

Defaulting to user installation because normal site-packages is not writeable
Collecting https://api.github.com/repos/jonescompneurolab/hnn-core/zipball/master
  Downloading https://api.github.com/repos/jonescompneurolab/hnn-core/zipball/master
     | 528.9 kB 3.2 MB/s 0:00:00
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Installing backend dependencies ... done
  Preparing metadata (pyproject.toml) ... done
Requirement already satisfied: numpy>=1.14 in /usr/lib/python3/dist-packages (from hnn-core==0.3.dev0) (1.21.5)
`

I also assumed that python would automatically be included in the developer version, yet I had to install it as well. 

Any thoughts?

**Comment by jasmainak** [comment]
@wagdy88 I'm not sure what exactly you did. Did you already have a conda environment with ipython installed in that environment and did you try to install `hnn-core` within it? There seems to be some problem with your set up because you get this message: "Defaulting to user installation because normal site-packages is not writeable". Our installation instructions say you need anaconda but it seems you are installing using the `pip` that comes with the system.

Python will not be "automatically included" ... you need to get it using conda. But numpy should get installed when you attempt to install the developer version.
